### PR TITLE
{bio}[foss/2022b] MACS3 v3.0.0b2

### PR DESCRIPTION
**Pull request by fizwit**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/m/MACS3/MACS3-3.0.0b2-foss-2021b.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+dependencies = [
+    ('Python', '3.9.6'),
+    ('SciPy-bundle', '2021.10'),
+    ('scikit-learn', '1.0.1'),
+    ('matplotlib', '3.4.3'),
+]
+
+use_pip = True
+
```
Comment (verdurin) [inline easybuild/easyconfigs/m/MACS3/MACS3-3.0.0b2-foss-2021b.eb:24]: @fizwit please add the missing `sanity_pip_check`.

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/m/MACS3/MACS3-3.0.0b2-foss-2021b.eb (hunk 1 above)
Comment (fizwit) [reply easybuild/easyconfigs/m/MACS3/MACS3-3.0.0b2-foss-2021b.eb:24]: MACS3 requires hmmlearn 0.3.0, which requires a new pybind11.  Need to switch to a newer toolchain.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3100.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/fac980abe74ad668003e092521c990d8 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @fizwit!

**Pull request merged**